### ISSUE TITLE
BFGS  vs sparse_cg

### ISSUE THREAD

**Issue by Sandy4321** [benjaminbluhm/regression_asymmetric_loss#2]
may you share , why you used BFGS solver
when for parse data 
scikit 
https://scikit-learn.org/stable/modules/generated/sklearn.linear_model.Ridge.html
uses 
‘sparse_cg’ uses the conjugate gradient solver as found in scipy.sparse.linalg.cg. As an iterative algorithm, this solver is more appropriate than ‘cholesky’ for large-scale data (possibility to set tol and max_iter)
